tests/general/CasingBearTest.py

```diff
@@ -0,0 +1,83 @@
+from queue import Queue
+from bears.general.CasingBear import CasingBear
+from tests.LocalBearTestHelper import LocalBearTestHelper
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.bears.LocalBear import LocalBear
+
+
+class CasingBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section("test section")
+        self.uut = CasingBear(self.section, Queue())
+
+    def test_defaults(self):
+        self.section.append(Setting("casing", "snake"))
+        self.section.append(Setting("language", "CPP"))
+        self.check_validity(self.uut, ["int abc_def;\n", "int ab_cd;\n"])
```

Comment:
so you need to test differently when you use dependency bears check my [PR](https://github.com/coala-analyzer/coala-bears/pull/468) for `indentationBear`  and see tests for assistance or tests for `CodeCloneDetectionBear`.